static/app/components/nav/sidebar.tsx

```diff
@@ -0,0 +1,220 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import Feature from 'sentry/components/acl/feature';
+import OrganizationAvatar from 'sentry/components/avatar/organizationAvatar';
+import Link from 'sentry/components/links/link';
+import {useNavContext} from 'sentry/components/nav/context';
+import Submenu from 'sentry/components/nav/submenu';
+import {useNavIndicator} from 'sentry/components/nav/useNavIndicator';
+import {
+  isNavItemActive,
+  isNonEmptyArray,
+  isSubmenuItemActive,
+  makeLocationDescriptorFromTo,
+  type NavSidebarItem,
+  resolveNavItemTo,
+} from 'sentry/components/nav/utils';
+import {space} from 'sentry/styles/space';
+import theme from 'sentry/utils/theme';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+
+function Sidebar() {
+  const organization = useOrganization();
+
+  return (
+    <Fragment>
+      <SidebarWrapper role="navigation" aria-label="Primary Navigation">
+        <SidebarHeader>
+          <OrganizationAvatar organization={organization} size={32} />
+        </SidebarHeader>
+        <SidebarItems />
+      </SidebarWrapper>
+      <Submenu />
+    </Fragment>
+  );
+}
+
+export default Sidebar;
+
+export function SidebarItems() {
+  const nav = useNavContext();
```

Comment:
nit: could destruct `config` from nav since it's the only item in use
```suggestion
  const { config } = useNavContext();
```
Only good reason for this is that it improves obfuscation passes.

Reply:
Nice one. Added in c0b98b054f0!